components/ui/filters.js

```diff
@@ -30,25 +29,13 @@ export default class Filters extends React.Component {
 
   setFilters(opts, key) {
     const newFilter = {};
-    newFilter[key] = isArray(opts) ? opts.map(o => o.value) : opts.value;
+    newFilter[key] = isArray(opts) ? opts.map(o => o) : [opts];
     const newFilters = Object.assign({}, this.props.selected, newFilter);
```

Comment:
Try to use the new syntax, if you want, of course... 
One const vs 2 const and an assignment

`const newFilters = {
  ...this.props.selected,
  [key]: isArray(opts) ? opts.map(o => o) : [opts]
}`